workflow/pred_temperature_transformer.ipynb

```diff
@@ -5,7 +5,7 @@
    "cell_type": "markdown",
```

Comment:
<p>This looks a lot better than the other notebook. Can you add some extra info with the figures? Some short analysis?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>I would prefer to do this later when we switch to the era5 data, which is longer than the example data. For now it is hard to say, especially given that the data we have is quite short and this behavior might not be robust.</p>